WordPress/src/main/res/values/strings.xml

```diff
@@ -3651,6 +3651,7 @@
     <string name="blogging_reminders_update">Update</string>
     <string name="blogging_reminders_done">Done</string>
     <string name="blogging_reminders_epilogue_title">All set!</string>
+    <string name="blogging_reminders_epilogue_not_set_title">Blogging reminders</string>
```

Comment:
Also, just to make sure you don't miss it, we have an updated spec for this string. Internal ref: pbArwn-2ox-p2